pegasus/emails/volunteer_contact_2015_receipt.md

```diff
@@ -7,29 +7,49 @@ end
 %>
 to: '<%= format_email_address(volunteer_email_s, volunteer_name_s) %>'
 from: '"Tanya Parker (Code.org)" <tanya_parker@code.org>'
+replyto: '<%= format_email_address(email_s, name_s) %>'
 subject: "A teacher is requesting your help for the Hour of Code"
 ---
 
 <% update_preferences = "http://#{CDO.canonical_hostname('code.org')}/volunteer/engineer/edit/#{volunteer_secret_s}/" %>
 
-### The following teacher is requesting your help for the Hour of Code
+Hi <%= volunteer_name_s %>,
 
-- **Name:** <%= name_s %>
-- **Email address:** [<%= email_s %>](<%= "mailto:" + email_s %>)
-- **School name:** <%= school_name_s %>
-- **School location:** <%= school_location_s %>
-- **Message:** <%= email_message_s %>
+<%= name_s %> is a teacher at <%= school_name_s %>. S/he found you on the Hour of Code site and after reviewing your profile specifically requested if you could help their class with the Hour of Code this year. S/he would like it if you could:
+
+***
+
+<% type_task_ss.each do |task| %>
+  <% if task == 'onsite' %>
+    * visit the classroom for technical help and inspiration
+  <% elsif task == 'remote' %>
+    * Skype into the classroom to say a few words of inspiration to the kids
+  <% elsif task == 'mentor' %>
+    * be a mentor to help prepare him/her for coding with his/her students 
+  <% end %>
+<% end %>
+  
+***
 
-Most schools still don’t teach computer science. Most students don’t know that it’s about solving big problems and connecting us all closer together. This is your chance to inspire at least one young person in your area. If you are available to volunteer with this teacher, **please contact the teacher directly with the above information**.
+We won't release your email directly to a teacher, so s/he's waiting for you to write back. 
+
+Most schools still don’t teach computer science. Most students don’t know that it’s about solving big problems and connecting us all closer together. This is your chance to inspire at least one young person in your area. If you are available to volunteer with this teacher, **please contact the teacher directly at [<%= email_s %>](<%= "mailto:" + email_s %>)**. 
 
 Tanya Parker<br>
 Product Manager, Code.org
 
+Contact information from teacher:
+- **Teacher Name:** <%= name_s %>
+- **Email address:** [<%= email_s %>](<%= "mailto:" + email_s %>)
+- **School name:** <%= school_name_s %>
+- **School location:** <%= school_location_s %>
+
 <hr/>
 
-Getting too many email requests?
+Getting too many email requests and/or already signed up to help a class?
```

Comment:
wasn't this supposed to say something like: "This means we don't have enough volunteers. Recruit your friends!"